packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/dataservice/ProjectController.java

```diff
@@ -181,7 +181,8 @@ public ResponseEntity<List<Project>> getProjects(
 			final Schema.Permission permission = projectService.checkPermissionCanWrite(
 					currentUserService.get().getId(), project.getId());
 
-			// Set the user permission for the project. If we are unable to get the user permission, we remove the
+			// Set the user permission for the project. If we are unable to get the user
+			// permission, we remove the
 			// project.
```

Comment:
Why is this shorter , but if it must be should we add "project" to the line above